source/api/discovery/0.9/index.md

```diff
@@ -121,7 +121,7 @@ Example Level 0 Activity:
 #### 2.1.2. Level 1: Resource Changes
 {: #level-1-basic-change-list}
 
-When dealing with large sets of resources, it can be useful to work with only those resources that have changed since the last time the list was processed. This can be facilitated by the addition of a time stamp that indicates when a resource was last modified or initially created. This is included using the `endTime` property, representing the time at which the activity of publishing the resource was finished. Lists with multiple activities are then ordered such that the most recent activities occur last. Consumers will then process the list of Activities in reverse order, from last to first, stopping when they encounter an Activity they have already processed.
+When dealing with large sets of resources, it can be useful to work with only those resources that have changed since the last time the list was processed. This is managed by the addition of a time stamp in the `endTime` property, that indicates the time at which the resource was last modified or initially created. Lists with multiple activities _MUST_ be ordered such that the most recent activities occur last. Consumers will then process the list of Activities in reverse order, from last to first, stopping when they encounter an Activity they have already processed.
```

Comment:
Not a blocker but I wonder whether _"...time at which the resource was last modified or initially created"_ might be better written as _"...time at which the resource was last modified (which could be its creation)"_

Reply:
I vaguely prefer calling both out separately, as there's `Update` and `Create` activity types which these allude to. Anyone want to provide a tie breaker opinion?

Reply:
I'd also have a preference for the text in the PR.

Reply:
I think the problem I have reading it is that one seems to be given a choice: use the lastmod or creation. When actually we mean use creation only if there have been no subsequent mods. How about: _"......time at which the resource was last modified or initially created (if there have been no subsequent updates)"_